mesheryctl/internal/cli/root/system/system.go

```diff
@@ -17,7 +17,7 @@ package system
 import (
 	"fmt"
 
-	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	config2 "github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
```

Comment:
"config2"... is there a conflict with using "config" as the package name?

Reply:
Nah it was fixed . This must be a leftover

Reply:
Just committed to fix this